README.md

```diff
@@ -288,6 +288,16 @@ You can run a single test or test class, just add the `android.testInstrumentati
 ./gradlew <Flavor><BuildType>executeScreenshotTests -Pandroid.testInstrumentationRunnerArguments.class=com.your.package.YourClassTest#yourTest
 ```
 
+## Using shot on API 29+
+
+Due to the new storage system implemented on API 29 and higher, Shot is unable to write the screenshots on the SD. Shot can't be used at the moment with apps whose ``Target SDK is >= 30`` over ``devices with API >= 30``.
```

Comment:
Can you specify the target sdk and device API limit has to be at the same time?

Reply:
I've tried to improve this part. I hope it's more clear now.